.pipelines/templates/run-unit-tests.yaml

```diff
@@ -37,13 +37,10 @@ stages:
         pool:
           name: "$(BUILD_POOL_NAME_DEFAULT_WINDOWS_ALT)"
         steps:
+          # Test changes under review; only run one go test per script
           - script: |
-              cd npm/
-              go test ./...
-              cd ../cni/
-              go test ./...
-              cd ../platform/
-              go test ./...
+              cd azure-container-networking/
+              go test ./npm/... ./cni/... ./platform/...
```

Comment:
I would add `-race` and `-timeout` as well.

Reply:
hmm I've tried adding -race but it seems to require CGO_ENABLED=1 (enabled), but on this particular node agent running the script (which is windows), it gives an error saying gcc is not found, making me believe this isn't possible at the moment, unless you have any other ideas of how to run -race? https://msazure.visualstudio.com/One/_build/results?buildId=108557696&view=logs&j=1e95c5bf-f785-5c1d-a4c1-33b038e882fb&t=4058cacc-5989-52ae-5341-5beff346fc28

Reply:
so if I'm understanding this correctly, the windows node at the moment doesn't have c compiler, and so we cannot have cgo enabled. In the future, we will need to somehow update the node agent to include gcc? In the meantime a `-timeout` has been added.

Reply:
No C compiler is required on the node for CGO_ENABLED=1. CGo means that some dependencies are dynamically linked (e.g. DNS resolution). As long as the DLLs for performing those functions are present on the system, it will work.